classes/class.tapestry-h5p.php

```diff
@@ -0,0 +1,19 @@
+<?php
+require_once dirname(__FILE__) . "/../interfaces/interface.tapestry-h5p.php";
+
+class TapestryH5P implements ITapestryH5P
+{
+  private $postId;
+
+  public function __construct($postId = null)
+  {
+    $this->postId = $postId;
+  }
+
+  public function get()
+  {
+    global $wpdb;
+    $content = $wpdb->get_results("select id, title from wp_h5p_contents");
```

Comment:
The table prefix `wp_` is not always the same on all sites, so please get the prefix dynamically using `$wpdb->prefix`.